app/src/main/res/layout/fragment_route_detail.xml

```diff
@@ -19,7 +19,15 @@
                 android:scaleType="centerCrop"
                 android:src="@drawable/ic_sample_layer"
                 android:background="@color/ic_sample_background_background"
-                tools:ignore="ContentDescription" />
+                tools:ignore="ContentDescription"
+                android:visibility="visible"/>
```

Comment:
Replace header with some loading indicator to give the user feedback that something is actually loading here.

Maybe one of these: https://github.com/ybq/Android-SpinKit  ?

Reply:
We should talk with Darja about this and maybe create a new Issue for the loading indicator.

Reply:
Oh sorry, looks easy with android spinkit, maybe we don't need another issue